.github/composite-actions/setup-php/action.yml

```diff
@@ -23,3 +23,4 @@ runs:
         coverage: ${{ inputs.coverage }}
         tools: composer, ${{ inputs.tools }}
         extensions: none, ctype, curl, dom, json, mbstring, opcache, openssl, simplexml, tokenizer, xml, xmlwriter, zip
+        ini-values: zend.assertions=1
```

Comment:
should we apply this to executing tests on local machine too ?

Reply:
By configuring Docker runtimes properly, yes 😉.

Reply:
in our own in-file repos, pls ?
also when one just trigger `composer sca` etc

Reply:
@keradus Composer scripts should be run via Docker Compose, so if our images are built with enabled assertions, tests will fail if assert is not matched. I wouldn't want hardcoded ini directives in Composer scripts.